src/js/stores/font.js

```diff
@@ -91,6 +91,98 @@ define(function (require, exports, module) {
 
             return fontObj && fontObj.postScriptName;
         },
+
+        getTypeObjectFromLayer: function (layer) {
+            if (layer.kind !== layer.layerKinds.TEXT) {
+                return null;
```

Comment:
If the caller always is prepared for `null` to be returned, then I'm not arguing that it should be an exception instead. Returning null is perfectly fine if the caller knows that it might get null. Just checking!